packages/worker/src/api/controllers/global/email.ts

```diff
@@ -14,6 +14,12 @@ export async function sendEmail(ctx: BBContext) {
     cc,
     bcc,
     automation,
+    addInvite,
```

Comment:
Same as other point, would be good if this was an invite structure with all invitation properties within - don't need the `addInvite` property if there is just a:
```
invite: {
  startTime,
  endTime,
  summary,
  location,
  url,
}
```

Can simply detect that the invite structure is present.